src/lib/iframe-tools/identity-manager.js

```diff
@@ -0,0 +1,65 @@
+/* @flow */
+import { setIdentity, getIdentity } from '../local-storage';
+import { getDeviceInfo } from '../get-device-info';
+import { logger } from '../logger';
+
+import { IframeManager } from './iframe-manager';
+
+export class IdentityManager extends IframeManager {
+  constructor(config) {
+    let iframeUrl;
+    
+    if (config.paramsToIdentityUrl) {
+      iframeUrl = config.paramsToIdentityUrl();
+    } else {
+      iframeUrl = 'https://www.paypal.com/muse/identity/index.html';
+    }
+
+    super({ src: iframeUrl });
+
+    this.addMessageListener(this.storeIdentity);
+  }
+
+  onIframeLoad = () => {
+    this.fetchIdentity();
+  }
+
+  logIframeError = (e) => {
```

Comment:
potential error here if `e` was undefined, or if `e.data` is null.
I think we should use lodash like `_.get`, or do `if` checks.